config/email_service.yml

```diff
@@ -1,8 +1,11 @@
 development:
   provider: <%= ENV["EMAIL_SERVICE_PROVIDER"] || "PSEUDO" %>
+  email_address_override:
```

Comment:
I wonder if these should also be configurable with environment variables?

Reply:
changed in 4a23d1cfb76349031d30739fe883ede788b6beb0